app/assets/javascripts/pulitzer.js

```diff
@@ -4,17 +4,35 @@ var Select2Trigger = Class.extend({
   }
 });
 
-$(document).ready(function(){
-  $(document).on('change', '#post_type_content_element_type_content_element_type_id', function(){
-    $dimensionsFields = $(this).parent().find($('.image_dimensions')).first();
-    if ($('option:selected', this).text() == "Image"){
-      $dimensionsFields.show();
-    } else {
-      $dimensionsFields.hide();
-    }
-  });
+var ToggleContentElementAttr = Class.extend({
```

Comment:
@gomayonqui, why is this done in javascript? Shouldn't there just be separate form views for each content element type? I think it would be much simpler. We would be able to completely remove this class.